closure/goog/testing/net/xhrio_test.js

```diff
@@ -55,6 +55,28 @@ function testStaticSend() {
   assertEquals('sendInstances_ after READY', 0, sendInstances.length);
 }
 
+function testStaticSendWithFactory() {
+  var xhrFactoryCalled = 0;
+  var optionsFactoryCalled = 0;
+
+  var wrapperFactory = new goog.net.WrapperXmlHttpFactory(
+      function() {
+        xhrFactoryCalled++;
+        return goog.net.XmlHttp();
```

Comment:
yeah, I also realised, creating an `XmlHttp` instance in this mock is quite un-necessary. So, now we simply return and empty object. I think that makes sense.